hearth/lib/hearth/http/fields.rb

```diff
@@ -107,9 +108,8 @@ def each(&block)
         end
         alias each_pair each
 
-        # @api private
         def inspect
-          to_h
+          to_h.to_s
```

Comment:
should this be to_h.inspect instead?

Reply:
Maybe but it should be the same value I think. Will change.